UPGRADE.md

```diff
@@ -10,9 +10,19 @@ The `autocompile` option is now the default option for the behavior of `\gregori
 
 ### Gregorio and Grana Padano fonts
 
-The gregorio and granapadano fonts, as well as their "-op" Dominican variants, will now be available as separate downloads.  If you were using these fonts, either download and install them or switch to greciliae, which continues to be distributed along with Gregorio by default.
+The Gregorio and Grana Padano fonts, as well as their "-op" Dominican variants, will now be available as a separate download.  If you were using these fonts, either download `supp_fonts-5_0_0.zip` and install them or switch to Greciliae, which continues to be distributed (with its "-op" Dominican variants) along with Gregorio by default.
```

Comment:
I didn't pick up on that.  Before switching it back, however, I had the following thought:

@eroux, is there some reason why Gregorio isn't the default font?  My understanding of its development was that it was the original font for the package and it was only later that the Caeciliae and Parmesan fonts were adapted to become Greciliae and Grana Padano.  Why was Greciliae made the default?

If my understanding of the history is correct, then logically it might make more sense to make Gregorio the default font (yes, I realize this will change lots of things) and then distribute Greciliae and Grana Padano as the supplemental fonts.  This might ease confusion as then "Gregorio" would always be referring to the defaults (though the font vs. program ambiguity would remain).

Another/additional option might be to rename the Gregorio font (like we did Parmesan -> Grana Padano) in order to eliminate the ambiguity.